src/index.ts

```diff
@@ -10,18 +10,37 @@
  */
 
 import * as ts from 'typescript';
+let argv = require('yargs')
+  .usage('Usage: <file> [options]')
+
+  .alias('l', 'noLines')
+  .describe('l', 'skip line numbers in output to keep it more diff-friendly')
+
+  .alias('v', 'verbose')
+  .describe('v', 'also print relevant code')
+
+  .argv;
 
 import checkFile from './checker';
 
-const [,, tsFile] = process.argv;
+const [tsFile] = argv._;
+const { noLines, verbose } = argv;
 
-// TODO: read options from a tsconfig.json file.
-const options: ts.CompilerOptions = {};
-const host = ts.createCompilerHost(options, true);
+// read options from a tsconfig.json file.
+let host = ts.createCompilerHost({}, true);
+const options: ts.CompilerOptions = ts.readConfigFile('tsconfig.json', (path: string) => host.readFile(path)).config['compilerOptions'];
```

Comment:
Does this mean that the tests now run with typings-checker's own `tsconfig.json`? That's probably fine.